vpr/src/route/route_timing.cpp

```diff
@@ -1035,25 +925,25 @@ int get_max_pins_per_net(const Netlist<>& net_list) {
 }
 
 template<typename ConnectionRouter>
-bool timing_driven_route_net(ConnectionRouter& router,
-                             const Netlist<>& net_list,
-                             ParentNetId net_id,
-                             int itry,
-                             float pres_fac,
-                             const t_router_opts& router_opts,
-                             CBRR& connections_inf,
-                             RouterStats& router_stats,
-                             std::vector<float>& pin_criticality,
-                             std::vector<vtr::optional<const RouteTreeNode&>>& rt_node_of_sink,
-                             float* net_delay,
-                             const ClusteredPinAtomPinsLookup& netlist_pin_lookup,
-                             std::shared_ptr<SetupHoldTimingInfo> timing_info,
-                             NetPinTimingInvalidator* pin_timing_invalidator,
-                             route_budgets& budgeting_inf,
-                             float worst_neg_slack,
-                             const RoutingPredictor& routing_predictor,
-                             const std::vector<std::unordered_map<RRNodeId, int>>& choking_spots,
-                             bool is_flat) {
+NetResultFlags timing_driven_route_net(ConnectionRouter& router,
```

Comment:
Comment. Some comment is part way in the function; move it up and fix return flags etc.